libminifi/src/controllers/SSLContextService.cpp

```diff
@@ -213,7 +213,8 @@ bool SSLContextService::addP12CertificateToSSLContext(SSL_CTX* ctx) const {
       core::logging::LOG_ERROR(logger_) << "Failed to set additional certificate from " << certificate_ << ", " << getLatestOpenSSLErrorString();
       return false;
     }
-    cacert.release();  // a successful SSL_CTX_add_extra_chain_cert() takes ownership of cacert
+    // a successful SSL_CTX_add_extra_chain_cert() takes ownership of cacert
+    cacert.release();  // NOLINT(bugprone-unused-return-value)
```

Comment:
Would casting the expression to void hide the warning? It might be an alternative to NOLINT.

Reply:
Good idea, updated in 3b4102f32349a842f90c11b08189d9e79411eca3